rmgpy/data/kinetics/family.py

```diff
@@ -1258,6 +1262,7 @@ def add_rules_from_training(self, thermo_database=None, train_indices=None):
                 reference=entry.reference,
                 short_desc="Rate rule generated from training reaction {0}. ".format(entry.index) + entry.short_desc,
                 long_desc="Rate rule generated from training reaction {0}. ".format(entry.index) + entry.long_desc,
+                metal=entry.metal
```

Comment:
Should this have `metal` and `facet` be separate like other database entries? Or are you expecting a label like `pt111` to be here? Would be nice to be consistent.